packages/sdk/src/credential/teamsUserCredential.browser.ts

```diff
@@ -236,6 +236,7 @@ export class TeamsUserCredential implements TokenCredential {
 
         const tokenResult: AccessTokenResult = response.data;
         const key = await this.getAccessTokenCacheKey(scopesStr);
+        // Important: tokens are stored in sessionStorage, read more here: https://aka.ms/teamsfx-session-storage-notice
```

Comment:
Can we also add this notice to document comments of TeamsUserCredential.getToken() function (as well as other possible places)?  So developers can see this notice when coding, as well as see it in our SDK reference doc.

Reply:
Sure, will add it

Reply:
Updated